src/extractors/mfcc.js

```diff
@@ -12,13 +12,29 @@ export default function(args) {
     throw new TypeError('Valid melFilterBank is required to generate MFCC');
   }
 
+  let numberOfMFCCCoefficients = 13;
+
+  if (args.numberOfMFCCCoefficients && typeof args.numberOfMFCCCoefficients === 'number') {
+    if (args.numberOfMFCCCoefficients > 40) {
+      numberOfMFCCCoefficients = 40;
+    } else if (args.numberOfMFCCCoefficients < 1) {
+      numberOfMFCCCoefficients = 1;
+    } else {
+      numberOfMFCCCoefficients = args.numberOfMFCCCoefficients;
```

Comment:
This could potentially be rewritten as:

```js
numberOfMFCCCoefficients = Math.min(40, Math.max(1, args.numberOfMFCCCoefficients));
```

Reply:
I had originally done it as a ternary expression: `(x => x < 0 ? 0 : x > 40 ? 40 : x)(args.numberOfMRCCCoefficients)`, but Nevo & I decided that was too ridiculous 😛 I'll switch to the `Math.min`/`Math.max` solution when I get time 😄